src/crimson/os/seastore/transaction_manager.h

```diff
@@ -579,6 +579,7 @@ class TransactionManager : public ExtentCallbackInterface {
 	      });
 	    } else {
 	      // absent
+	      cache->retire_absent_extent_addr(t, original_paddr, original_len);
```

Comment:
The cache index and the lba tree linkage should be consistent.

The created retired placeholder is added to the cache index, it needs to be linked in the **stable** lba tree as well.

Reply:
Leaving it to https://tracker.ceph.com/issues/71363